spec/support/fixtures/ci/github.json

```diff
@@ -710,5 +710,70 @@
       "git.commit.sha": "b9f0fb3fdbb94c9d24b2c75b49663122a529e123",
       "git.repository_url": "https://1.1.1.1:1234/ghactions-repo.git"
     }
+  ],
+  [
```

Comment:
here are the actual test cases:
1. happy path
2. case where events file is not found